assets/pages/bundles-landing/components/Introduction.jsx

```diff
@@ -3,19 +3,53 @@
 // ----- Imports ----- //
 
 import React from 'react';
+import type { Participations } from 'helpers/abtest';
+
+/* eslint-disable react/no-unused-prop-types */
+type PropTypes = {
+    abTests: Participations,
+}
```

Comment:
I would add a comment here to enable the linter back, if not that rule will be disabled for the entire file.

Reply:
ah nice spot, I removed the comment and we didn't need it